crates/stdx/src/lib.rs

```diff
@@ -200,6 +201,13 @@ impl Drop for JodChild {
     }
 }
 
+impl JodChild {
+    pub fn into_inner(self) -> process::Child {
+        // SAFETY: repr transparent
+        unsafe { std::mem::transmute::<JodChild, process::Child>(self) }
```

Comment:
Just wondering, why not `self.0`?

Reply:
It implements `Drop`, so you can't move out of it